decidim-core/app/views/decidim/notification_mailer/event_received.html.erb

```diff
@@ -9,13 +9,30 @@
 <% end %>
 
 <% if @event_instance.try(:safe_resource_text).present? %>
+  <% if @event_instance.perform_translation? %>
+    <p style="font-weight: bold"><%= t(".original_text") %></p>
+  <% end %>
+
   <blockquote>
     <p>
       <%= @event_instance.safe_resource_text %>
     </p>
   </blockquote>
 <% end %>
 
+<% if @event_instance.content_in_same_language? %>
```

Comment:
To be confirmed with @carolromero: should we show the "The content has been posted in your preferred language (%{language}), this is why no automated translation is displayed in this email." text in the email?

Reply:
@andreslucena agreed.

Reply:
@carolromero it is not clear if the "The content has been posted in your preferred language (%{language}), this is why no automated translation is displayed in this email." should be left there or not.

Reply:
yes, it should be.

Reply:
thanks @carolromero , This is the implemented behavior, so i will mark the conversation as resolved.